.github/workflows/extension-attach-artifact-release.yml

```
@@ -155,42 +155,53 @@ jobs:
+          EXTRA_LINUX_COMMAND: ${{ inputs.extraLinuxCommand }}
         run: |
-          ${{ inputs.extraLinuxCommand }}
+          ${EXTRA_LINUX_COMMAND}
 
       - name: Build and Package
         if: ${{ inputs.dry_run == false }}
         shell: bash
         env:
           MAVEN_PROFILES: ${{ inputs.mavenProfiles }}
+          EXTRA_MAVEN_ARGS: ${{ inputs.extraMavenArgs }}
+          LIQUIBASE_VERSION: ${{ inputs.liquibaseVersion }}
```

actionlint (1.7.9)

property "liquibaseversion" is not defined in object type {artifactid: string; artifactpath: string; combinejars: bool; dry_run: bool; dry_run_version: string; extracommand: string; extramavenargs: string; javabuildversion: string; mavenprofiles: string; os: string; zip: string}

(expression)